.github/workflows/run-tests.yml

```diff
@@ -1,5 +1,7 @@
 name: All Tests
 
+run-name: ${{ inputs.test_name }}${{ inputs.test_dbs }}${{ inputs.run_single_test_job }}
```

Comment:
Does this need any delimiters to make it readable? How does it affect tests that aren't run in "single" mode?